src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/LanguageServices/CSharpRemoveUnnecessaryImportsService.Rewriter.cs

```diff
@@ -57,7 +59,7 @@ public override SyntaxNode DefaultVisit(SyntaxNode node)
                         currentUsings[i] = null;
 
                         var leadingTrivia = currentUsing.GetLeadingTrivia();
-                        if (ShouldPreserveTrivia(leadingTrivia))
+                        if (ShouldPreserveTrivia(leadingTrivia) || (i == 0 && preserveTriviaFromFirstUsing))
```

Comment:
do we need to pass in this bool?  coudl we not just check usings[i].Parent?

Reply:
Felt a bit nicer to keep this method just dealing with a list of usings, but I'm not too fussed. Will change